src/components/organisms/AssetsList.tsx

```diff
@@ -75,73 +69,90 @@ const AssetssList = ({ data }: Props) => {
     },
     {
       title: "Qty",
-      dataIndex: "qty",
       key: "qty",
       width: 200,
-      render: (quantity: string) => (
-        <QuantityDisplay value={parseFloat(quantity)} />
-      ),
+      render: (_, { quantity }) => <QuantityDisplay value={quantity} />,
     },
     {
       title: "Entry price",
       dataIndex: "",
-      key: "avg_entry_price",
+      key: "avgEntryPrice",
       width: 200,
-      render: ({ avg_entry_price }: any) => (
-        <PriceDisplay value={parseFloat(avg_entry_price)} />
-      ),
+      render: (_, { avgEntryPrice }) => <PriceDisplay value={avgEntryPrice} />,
     },
     {
       title: "Current price",
-      dataIndex: "",
       key: "current_price",
       width: 200,
-      render: ({ current_price }: any) => (
-        <PriceDisplay value={parseFloat(current_price)} />
+      render: (_, { currentPrice }) => <PriceDisplay value={currentPrice} />,
+    },
+    {
+      title: "MA",
+      key: "ma",
+      width: 130,
+      render: (_, { movingAvg }) => (
+        <PriceDisplay value={movingAvg}></PriceDisplay>
       ),
     },
     {
       title: "Change today",
       dataIndex: "change_today",
       width: 200,
-      render: (change_today: any) => (
+      render: (_, { changeToday }) => (
         <PercentageDisplay
           indicator={
-            change_today > 0
+            changeToday > 0
               ? INDICATOR.POSITIVE
-              : change_today === 0
+              : changeToday === 0
               ? INDICATOR.NEUTRAL
               : INDICATOR.NEGATIVE
           }
-          value={parseFloat(change_today) * 100}
+          value={changeToday * 100}
         />
       ),
     },
     {
       title: "Change since entry",
-      dataIndex: "change_since_entry",
+      key: "change_since_entry",
       width: 200,
-      render: (change_since_entry: any) => (
+      render: (_, { changeSinceEntry }) => (
         <PercentageDisplay
           indicator={
-            change_since_entry > 0
+            changeSinceEntry > 0
               ? INDICATOR.POSITIVE
-              : change_since_entry === 0
+              : changeSinceEntry === 0
               ? INDICATOR.NEUTRAL
               : INDICATOR.NEGATIVE
           }
-          value={parseFloat(change_since_entry) * 100}
+          value={changeSinceEntry * 100}
         />
       ),
     },
+
+    {
+      title: "Stop loss",
+      key: "stoploss",
+      render: (_, { stopLossType }) => stopLossType,
+    },
+    {
+      title: "SL Price",
+      key: "stoploss_price",
+      render: (_, { stopLossPrice }) => {
+        return <PriceDisplay value={stopLossPrice}></PriceDisplay>;
+      },
+    },
+    {
+      title: "Profit taken",
+      key: "partial_profit_taken",
+      render: (_, { takenPartialProfit }) => (takenPartialProfit ? "✅" : ""),
```

Comment:
Typiskt "Generation Z" att kötta in "Wingdings" rätt in i koden då här! :') 

(I like)

Reply:
Haha det är cooolt ju!